### PR TITLE
fix e2e env clean and update doc.

### PR DESCRIPTION
**Pull request by morvencao**
Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

**Comment by morvencao** [comment]
/hold

Looks like the hub and spoke operator from operator hub is not the latest one, so there will be no some new features.

File: README.md [hunk 1]
```diff
@@ -50,30 +50,31 @@ The supported component images include the following **keywords**:
...
-Before you run the E2E, make sure [ginkgo](https://github.com/onsi/ginkgo) is installed:
+1. Before you run the E2E, make sure [ginkgo](https://github.com/onsi/ginkgo) is installed:
 
 ```
-$ go get -u github.com/onsi/ginkgo/ginkgo
+go get -u github.com/onsi/ginkgo/ginkgo
 ```
 
-then, depending on how your E2E environment is set up, you can choose the following exclusive methods to run the e2e testing:
+2. then, depending on how your E2E environment is set up, **choose the following exclusive methods to run the e2e testing**:
```
Comment (songleo) [inline README.md:65]: then -> Then

Comment (morvencao) [reply README.md]: done

**Review by songleo — APPROVED** [review]

**Pull request merged**